js/src/offcanvas.js

```diff
@@ -121,6 +121,11 @@ class Offcanvas extends BaseComponent {
     this._element.classList.add(CLASS_NAME_SHOW)
 
     const completeCallBack = () => {
+      if (!this._config.scroll) {
+        this._focustrap.activate()
```

Comment:
now that both focusTrap activations are followed by element focus, you may consider to move it inside activation 👼

Reply:
Yeah - I had considered that. My hesitation was that it assumes that the `trapElement` itself actually can be focused (tabindex > -2). So, there would need to be additional checks done in the focus trap class before trying to focus itself. Your call, I'm cool with it either way.

Reply:
As both components doesn't check something like this, I think you can just add an option `autofocus|focusOnInit` that will be true by default and make it act accordingly